lib/entry.js

```diff
@@ -74,17 +74,19 @@ function SalesforceEntity(callScope) {
    */
   this.processTrigger = async function processTrigger(msg, cfg, snapshot) {
 
-    if (!snapshot || typeof snapshot !== 'string') {
-      // Empty snapshot
-      console.warn('Warning: expected snapshot but found: ',
-          snapshot===''?'empty string':snapshot);
-      snapshot = new Date('1970-01-01T00:00:00.000Z').toISOString();
+    if (!snapshot || (typeof(snapshot) === 'object' && !snapshot.previousLastModified)) {
+      snapshot = {
+        previousLastModified: cfg.startTime || new Date('1970-01-01T00:00:00.000Z').toISOString(),
```

Comment:
`new Date('1970-01-01T00:00:00.000Z').toISOString()` can be simplified to `'1970-01-01T00:00:00.000Z'`

Reply:
Good point, done.